### ISSUE TITLE
decimal number comparison

### ISSUE THREAD

**Issue by nikarh** [lukas-krecan/JsonUnit#130]
Running the following test yields an assertion error:
```java
public class JsonTest {

    @Test
    public void assertJson() {
        assertThatJson("[{\"b\": 10}]")
            .isArray()
            .isEqualTo(JsonAssertions.json"[{\"b\": 10}]"));
    }

}
```

Expected test to pass. Got the following error instead:
```
java.lang.AssertionError: JSON documents are different:
Different value found in node "[0].b", expected: <10> but was: <1E+1>.
```

**Comment by lukas-krecan** [comment]
Hi, thanks for feedback. It's indeed strange, I will take a look at it, in the meantime you can use this
```

        assertThatJson("[{\"b\": 10}]")
            .withConfiguration(c -> c.withTolerance(0))
            .isArray()
            .isEqualTo(json("[{\"b\": 10}]"));
```

**Comment by lukas-krecan** [comment]
Fixed in 2.0.0.RC2
